Library/Homebrew/test/caveats_spec.rb

```diff
@@ -26,4 +26,39 @@ def caveats
       expect(described_class.new(f)).not_to be_empty
     end
   end
+
+  describe "#caveats" do
+    describe "when f.plist is not null" do
```

Comment:
I wanted to stub methods received by `f` and these can't be stubbed inside `context` / `describe` blocks, hence, I chose `it`. I'll use `context` when using `when` from  now on.